src/reportWebVitals.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable object-curly-newline */
```

Comment:
- [ ] `required`
- [ ] Customizing the Linters by `Disabling the eslint` defeats the purpose of using it please enable it.

Reply:
@mwanawabangona this /* eslint-disable object-curly-newline */ is from react I didn't write that code so I think it is ok.

Reply:
And thanks for the comments :) but in this case \i don't know what else I can do

Reply:
@peterrobert.... did you read what I wrote to the last reviewer. What do you want me to do I will get a lintor error if I import or don't import react. The other was about arrow functions did you look at my arrow functions are they ok or do I have to change them?  The other error was from the code from react. Should I change that code it was from react ?

Reply:
@peterrobert should i send it again it again for a third review or what should I change?

Reply:
@peterrobert there is nothing to change

Reply:
@david63011  give me a min... Let me find solutions for you online

Reply:
@david63011 honestly I don't get why you are getting these errors. I can see most of your work is okey. Did you make any changes to the linter config?